src/views/Login.vue

```diff
@@ -92,9 +94,15 @@ export default class Login extends Vue {
           // Don't redirect automatically; we handle that above.
           return false;
         },
+        uiShown: () => this.recordEvent('loginShown'),
```

Comment:
Should 'loginShown' be in a file of constants?

Reply:
no; it's only used in one place and is intended for human consumption: i want someone who sees 'loginShown' in a log to be able to grep for the string and end up here.

Reply:
Ok